hca/__init__.py

```diff
@@ -2,13 +2,13 @@
 
 import os, logging
 
-import tweak
+from tweak import Config as _Config
```

Comment:
why this change?

Reply:
I'm importing it with an underscore because the Config object should not be part of this package's API.

Reply:
Ah!  Makes sense.